src/avkit.cs

```diff
@@ -62,18 +64,31 @@ interface AVPictureInPictureController
 		[Export ("pictureInPictureSuspended")]
 		bool PictureInPictureSuspended { [Bind ("isPictureInPictureSuspended")] get; }
 
+		[iOS (13, 0)]
+		[Static]
+		[Export ("pictureInPictureButtonStartImage")]
+		UIImage PictureInPictureButtonStartImage { get; }
+		
+		[iOS (13, 0)]
+		[Static]
+		[Export ("pictureInPictureButtonStopImage")]
+		UIImage PictureInPictureButtonStopImage { get; }
 		
+#if !MONOMAC
 		[Static]
 		[Export ("pictureInPictureButtonStartImageCompatibleWithTraitCollection:")]
 		UIImage CreateStartButton ([NullAllowed] UITraitCollection traitCollection);
 
 		[Static]
 		[Export ("pictureInPictureButtonStopImageCompatibleWithTraitCollection:")]
 		UIImage CreateStopButton ([NullAllowed] UITraitCollection traitCollection);
+#endif
 	}
+	
+	interface IAVPictureInPictureControllerDelegate {}
 
-	[NoTV]
-	[Protocol, Model]
+	[iOS (9,0), Mac (10,15), NoTV]
+	[Protocol, Model (AutoGeneratedName = true)]
```

Comment:
Setting `AutoGeneratedName` to true when it previously wasn't is a breaking change.